tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2015",
     "strict": true,
     "lib": ["es2020", "DOM"],
-    "module": "commonjs",
+    "module": "Node16",
```

Comment:
If we change this, we can not pass the CI with Node14. But can we drop it?
Svelte4 need at least Node16.
(Then we can merge https://github.com/sveltejs/svelte-eslint-parser/pull/399)

Reply:
Oh, I did not know that. I'll revert it.